SpatialGDK/Source/SpatialGDK/Private/Interop/ClientNetLoadActorHelper.cpp

```diff
@@ -0,0 +1,142 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Interop/ClientNetLoadActorHelper.h"
+
+#include "Algo/AnyOf.h"
+#include "EngineClasses/SpatialNetDriver.h"
+#include "Interop/ActorSystem.h"
+#include "Interop/SpatialReceiver.h"
+#include "Interop/SpatialSender.h"
+#include "Schema/Restricted.h"
+#include "SpatialConstants.h"
+#include "Utils/RepLayoutUtils.h"
+#include "Utils/SpatialActorUtils.h"
+
+DEFINE_LOG_CATEGORY(LogClientNetLoadActorHelper);
+
+namespace SpatialGDK
+{
+FClientNetLoadActorHelper::FClientNetLoadActorHelper(USpatialNetDriver& InNetDriver)
+{
+	NetDriver = &InNetDriver;
+}
+
+UObject* FClientNetLoadActorHelper::GetReusableDynamicSubObject(const FUnrealObjectRef ObjectRef)
+{
+	if (FNetworkGUID* SubObjectNetGUID = GetSavedDynamicSubObjectNetGUID(ObjectRef))
+	{
+		if (UObject* DynamicSubObject = NetDriver->PackageMap->GetObjectFromNetGUID(*SubObjectNetGUID, false))
+		{
+			NetDriver->PackageMap->ResolveSubobject(DynamicSubObject, ObjectRef);
```

Comment:
i copied this in from actorsystem, but is this really needed? from what I can see, resolvesubobject gives the subobj a net guid if it doesn't have one - but at this point in time, it's guaranteed to have a netguid.

I don't want to just make the change incase I'm missing some nuance about `ResolveSubobject` though

Reply:
I think it may still be needed, as this will also associate that NetGuid with the given ObjectRef